tests/UI/pages/BO/advancedParameters/import/index.js

```diff
@@ -28,5 +39,54 @@ class Import extends BOBasePage {
     ]);
     return download.path();
   }
+
+  /**
+   * Select the type of the file and upload the sample file
+   * @param page
+   * @param dropdownValue
+   * @param filePath
+   * @return {Promise<string>}
+   */
+  async uploadSampleFile(page, dropdownValue, filePath) {
+    await this.selectByVisibleText(page, this.fileTypeSelector, dropdownValue);
+    await page.setInputFiles(this.fileInputField, filePath);
+
+    return this.getTextContent(page, this.alertSuccessBlock);
+  }
+
+  /**
+   * Click on the "Next step" button
```

Comment:
```suggestion
   * Go to 'Next step' of import
```

Can you use single quote instead ?